### ISSUE TITLE
Could not create vmnet interface

### ISSUE THREAD

**Issue by dweidenfeld** [nlf/dlite#43]
```
$ sudo dlite daemon
virtio_net: Could not create vmnet interface, permission denied or no entitlement?
```

do you have any idea what cause this error? (Running OS X 10.11.3)

**Comment by nlf** [comment]
can you show me the contents of `~/.dlite/config.json`? the only issue i've ever seen that i can solve on my end with this is if somehow you ended up with an invalid uuid

**Comment by dweidenfeld** [comment]
sure

```
{"uuid":"483d0b78-c474-11e5-971c-acbc329bcfdb","cpu_count":1,"memory":1}
```

should I remove the .dlite folder and give it another try?!

**Comment by dweidenfeld** [comment]
Okay, now I deleted the .dlite folder and the initiation process seems to work.

So if I run `sudo dlite daemon` everything works as intended.
But if I run `launchctl start local.dlite` it creates the /var/run/docker.sock file, but a `docker info` just hangs.

Any ideas?

**Comment by nlf** [comment]
check your `/etc/hosts` file and make sure there is an entry for local.docker and that the IP is reachable. it's likely that either the wrong IP is being detected or the machine isn't actually booting.

you can also check the Console app for errors, sometimes it displays things that are difficult to notice otherwise

**Comment by dweidenfeld** [comment]
Okay now everything works. Seems that I had to uninstall everything, remove the .dlite folder and the hosts entry.
After that I did a fresh install, but stuck with the "hanging" problem.
After a reboot of OS X everything worked.

Thanks for your support ;)

PS: I did the ping while it hangs, and the IP was specified correct, so the host answered. I think the problem was related to my ssh keys or something like that.
